packages/app/src/applications/multi-instance-configuration/webui/index.tsx

```diff
@@ -11,9 +13,14 @@ const params = getSearchParams();
 const manifestURL = params.get('manifestURL')!;
 const applicationId = params.get('applicationId')!;
 
+const themeColorsObservable = new Subject<any>();
+window.bx.theme.addThemeColorsChangeListener(
+  (_: any, result: any) => themeColorsObservable.next(result)
+);
```

Comment:
We'd probably need a generic helper or wrapper for this kind of pattern

Reply:
hmm...
it's the only place I use it.

Reply:
I would find it more readable to have a dedicated function to turn contextBridge listeners to Observable.
It's more for future potential use than anything else, so do as you wish :)